src/js_subscriptions.hpp

```diff
@@ -139,13 +139,13 @@ template <typename T>
 void SubscriptionClass<T>::get_name(ContextType ctx, ObjectType this_object, ReturnValue& return_value)
 {
     auto sub = get_internal<T, SubscriptionClass<T>>(ctx, this_object);
-    auto name = sub->name();
+    auto name = sub->name;
```

Comment:
```suggestion
    const auto& name = sub->name;
```

No need to force an extra copy. Or just use `sub->name` directly everywhere.